### PR TITLE
Add accounts to redux state

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch7109]

File: src/api/db.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const init = (password: string) => {
   try {
     store = new Store({ encryptionKey: password, clearInvalidConfig: true });
     // Write something to the store to actually create the file
-    store.set('accounts', []);
+    store.set('accounts', {});
```
Comment (Mrtenz) [inline src/api/db.ts:15]: Wouldn't this override the accounts every time `init` is called? Maybe check if `accounts` exists before writing?

File: src/app/store/persistor.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { EnhancedStore } from '@reduxjs/toolkit';
+import { persistStore } from 'redux-persist';
+
+import { ApplicationState } from '.';
+
+export const createPersistor = (store: EnhancedStore<ApplicationState>) =>
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```
Comment (Mrtenz) [inline src/app/store/persistor.ts:8]: Why?

File: src/app/store/utils/storage.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { getAccounts, setAccounts } from '@app/services';
+
+export const storage = {
+  getItem: (key: string) => {
+    if (key === 'accounts') {
+      return getAccounts();
+    }
+    throw new Error('Invalid key');
+  },
+  setItem: (key: string, value: any) => {
```
Comment (Mrtenz) [inline src/app/store/utils/storage.ts:10]: Can you replace `any` with `unknown` here?

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: src/api/db.ts (hunk 1 above)
Comment (FrederikBolding) [reply src/api/db.ts:15]: `init` is only for initializing the store and will delete anything in the store anyway!

File: src/app/store/persistor.ts (hunk 2 above)
Comment (FrederikBolding) [reply src/app/store/persistor.ts:8]: For some reason the `redux-persist` typing doesn't include `manualPersist`

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**